vector/src/main/java/im/vector/app/features/reactions/EmojiRecyclerAdapter.kt

```diff
@@ -43,13 +44,13 @@ import kotlin.math.abs
  * TODO: Performances
```

Comment:
Remove this TODO?

Reply:
It doesn't really improve performance on this screen, so not really

Reply:
ok